src/platforms/common/configuration/releases.mdx

```diff
@@ -46,26 +48,38 @@ The value can be arbitrary, but we recommend either of these naming strategies:
+- **Commit SHA**: If you use a version control system like Git, we recommend using the identifying hash (for example, the commit SHA, `da39a3ee5e6b4b0d3255bfef95601890afd80709`). You can let Sentry CLI automatically determine this hash for supported version control systems. See [Sentry CLI Documentation](/product/cli/releases/#creating-releases).
 
 <Note>
 
 Releases are global per organization; prefix them with something project-specific for easy differentiation.
 
 </Note>
 
-<PlatformContent includePath="set-release" notateUnsupported />
+## Setting a release
+
+How you make the release name (or version) available to your code is up to you. For example, you could use an environment variable that is set during the build process or during initial start-up.
 
-How you make the version available to your code is up to you. For example, you could use an environment variable that is set during the build process.
+Setting the release name tags each event with that release name. We recommend that you tell Sentry about a new release before sending events with that release name, as this will unlock a few more features. See our documentation about [releases](/product/releases/).
 
-This tags each event with the release value. We recommend that you tell Sentry about a new release before deploying it, as this will unlock a few more features as discussed in our documentation about [releases](/product/releases/). But if you don’t, Sentry will automatically create a release entity in the system the first time it sees an event with that release ID.
+If you do not tell Sentry about a new release, Sentry will automatically create a release entity in the system the first time it sees an event with that release ID.
 
 After configuring your SDK, you can install a repository integration or manually supply Sentry with your own commit metadata. Read our documentation about [setting up releases](/product/releases/setup/) for further information about integrations, associating commits, and telling Sentry when deploying releases.
 
 <PlatformSection supported={["apple", "android", "javascript", "flutter", "native", "rust", "python", "react-native", "dotnet", "unity"]}>
 
 ## Release Health
-  
+
+### Sessions
+
+A session represents the interaction between the user and the application. Sessions contain a timestamp, a duration, a status (if the session was OK or if it crashed) and others.
+
+Sessions are always linked to a release and are used to monitor user adoption, usage of the application, percentage of [crashes](/product/releases/health/#crash), and [session data](/product/releases/health/#session) of that release.
+
+This will give you insight into the impact of crashes and bugs as it relates to user experience, and reveal trends with each new issue through the release details, graphs, and filters.
```

Comment:
Connecting these two thoughts into one paragraph or the second part becomes a bit confusing in terms of what "this" is referring to. 

Which graphs and filters are you specifically talking about here? The ones on the Release Details page?

```suggestion
Sessions are always linked to a release and are used to monitor user adoption, usage of the application, percentage of [crashes](/product/releases/health/#crash), and [session data](/product/releases/health/#session) of that release. This will give you insight into the impact of crashes and bugs on user experience and reveal trends with each new issue through the release details, graphs, and filters.
```